packages/terra-site/src/examples/alert/InfoExample.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Alert from 'terra-alert';
+
+const alertStartTagStr = '<Alert type={Alert.Types.INFO} >';
+const alertEndTagStr = '</Alert>';
+const alertText = 'This is an information alert';
+
+const InfoExample = () => (
+  <div>
+    <div dir="ltr">
+      <h3>Basic Alert of type info with default title</h3>
+      <br />
+      <code>
```

Comment:
I'm fine with this, we'd like to eventually display code examples with the rendered components via a component. Here is the related issue logged for this work: https://github.com/cerner/terra-core/issues/117